app/src/main/res/layout/random_preset_fragment.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
would be a good idea to wrap the form in a scroll view. It should be required during landscape orientation.

Also, why didn't you just rename `app/src/main/res/layout/dialog_fragment__random_preset.xml` to `app/src/main/res/layout/random_preset_fragment.xml`?

Reply:
I don't know, didn't think about that simple thing lol, my bad there